Code-Of-Conduct.rst

```diff
@@ -0,0 +1,8 @@
+====================================================================
+package_template_build Contributor Covenant Code of Conduct
+====================================================================
+
+The PlanetaryPy Project Code of Conduct][Conduct] applies to
+package_template_build.
```

Comment:
something wrong with Markdown links here

Reply:
Yup.

Reply:
I'd make the link text "Code of Conduct" at the very least.

Reply:
done